.travis.yml

```diff
@@ -159,7 +159,20 @@ script:
         openssl aes-256-cbc -K $encrypted_675f1a0c317c_key -iv $encrypted_675f1a0c317c_iv -in ./kivy/tools/travis/id_rsa.enc -out ~/.ssh/id_rsa -d;
         chmod 600 ~/.ssh/id_rsa;
         echo -e "Host $SERVER_IP\n\tStrictHostKeyChecking no\n" >> ~/.ssh/config;
-        rsync -avh -e "ssh -p 2458" --include="*/" --include="*manylinux*.whl" --exclude="*" "wheelhouse/" "root@$SERVER_IP:/web/downloads/ci/linux/kivy/";
+        rsync -avh -e "ssh -p 2458" --include="*/" --include="*manylinux*.whl" --include="*armv7l*.whl" --exclude="*" "wheelhouse/" "root@$SERVER_IP:/web/downloads/ci/linux/kivy/";
+      elif [ "${RUN}" == "wheels" ]; then
+        # Cross compile the wheel for armv7l (Raspberry Pi 4)
+        docker build -f .ci/Dockerfile.armv7l -t kivy/kivy-armv7l --build-arg APP_IMAGE="balenalib/armv7hf-debian:buster" --build-arg CYTHON_INSTALL="$CYTHON_INSTALL" .;
+        CONTAINER_ID=$(docker create kivy/kivy-armv7l);
+        docker cp "${CONTAINER_ID}":/kivy-wheel .;
+        cp kivy-wheel/Kivy-* wheelhouse/;
+        for name in wheelhouse/*armv7l*.whl; do
+          new_name="${name/$tag_name-/$wheel_name}";
+          if [ ! -f "$new_name" ]; then
+            cp -n $name $new_name;
+          fi;
+        done;
+        ls wheelhouse/;
```

Comment:
Btw I intend to move this code up into the code above, but I just put it here, so it would get run by Travis.